DefInjected/FactionDef/Factions_Misc.xml

```diff
@@ -14,7 +14,7 @@
 	<Pirate.leaderTitle>капитан пиратов</Pirate.leaderTitle>
 
 	<Tribe.label>племя</Tribe.label>
-	<Tribe.description>Эти люди обитают на планете очень давно. Возможно, их предки разбились здесь тысячи лет назад. А может они пережили катаклизм, который разрушил более развитую цивилизацию. В любом случае, большинство аборигенов — кочевники, живущие за счёт земли и использующие примитивное оружие и орудия труда.\n\nОдни племена более агрессивны, другие открыты для торговли и альянсов, даже со странными народами вроде вас. Но близкое сотрудничество или найм будут сильно затруднены колоссальным социо-технологическим и языковым барьером между аборигенами и вами.\n\nНесмотря на очевидную технологическую отсталость, дикари могут быть опасными врагами и ценными союзниками, благодаря своей выносливости, числу и навыкам обращения с примитивным оружием.</Tribe.description>
+	<Tribe.description>Эти люди обитают на планете очень давно. Возможно, их предки разбились здесь тысячи лет назад. А может они пережили катаклизм, который разрушил более развитую цивилизацию. В любом случае, большинство аборигенов — кочевники, живущие за счёт земли и использующие примитивное оружие и орудия труда.\n\nОдни племена более агрессивны, другие открыты для торговли и альянсов, даже со странными народами вроде вас. Но близкое сотрудничество или найм будут сильно затруднены колоссальным социотехнологическим и языковым барьером между аборигенами и вами.\n\nНесмотря на очевидную технологическую отсталость, дикари могут быть опасными врагами и ценными союзниками, благодаря своей выносливости, числу и навыкам обращения с примитивным оружием.</Tribe.description>
```

Comment:
`These people have been here a very long time. Maybe their ancestors crashed here a thousand years ago. Maybe they survived some cataclysm that destroyed a technological civilization here. In any case, the tribals are mostly nomadic people who live off the land using primitive tools and weapons.\n\nSome tribes are more aggressive than others; many are open to trade and alliances, even with strange peoples like yourselves. Close cooperation or recruitment will be very difficult because of the vast sociotechnological and linguistic gap between the tribals and yourselves.\n\nDespite their apparent technological weakness, the tribals can be dangerous enemies and valuable friends because of their skill with low-tech warfare, their numbers, and their hardiness`